src/openms/include/OpenMS/ANALYSIS/ID/BayesianProteinInferenceAlgorithm.h

```diff
@@ -131,7 +131,7 @@ namespace OpenMS
     void inferPosteriorProbabilities(
         ConsensusMap& cmap,
         bool greedy_group_resolution,
-        std::optional<const ExperimentalDesign> exp_des = std::optional<const ExperimentalDesign>());
+        const std::optional<const ExperimentalDesign>& exp_des = std::optional<const ExperimentalDesign>());
```

Comment:
```suggestion
        const std::optional<const ExperimentalDesign>exp_des;
```
turns out, only those two :)

Reply:
great! thanks. I will check. Maybe there is a clang-tidy fix for this as well :)